### ISSUE TITLE
Wrong download url for recuva

### ISSUE THREAD

**Issue by pitsi** [jivkok/Chocolatey-AutomaticPackages#22]
I tried installing recuva via chocolatey yesterday and it failed completely. 
As it seems, the url your package provides, i.e.
`https://download.piriform.com/rcsetup153.exe`
leads to an "access denied" error.

The new and working url is now
`https://download.ccleaner.com/rcsetup153.exe`
so please correct it when possible.

**Comment by Destroy666x** [comment]
Is this maintained? Still an issue.

**Comment by pitsi** [comment]
Do you think I should file an issue in chocolatey itself and ask for its complete removal?

**Comment by Destroy666x** [comment]
I don't think this will do anything. I contacted them about removing stuff like Flash which is not only outdated when it comes to packages but also completely unsupported, yet they didn't do anything. I'm pretty sure that at least a replacement package/repo is needed.

**Comment by asheroto** [comment]
@jivkok are you still maintaining Chocolatey packages? Recuva is still broken and the repo hasn't been updated since 2016. You can add me, same username, on Chocolatey and I can assist in maintaining this.
